posts/deployToEcsWithGithubAction.md

```diff
@@ -0,0 +1,177 @@
+---
+authors:
+- Amine Ouali Alami
+tags:
+- Spring Boot
+- JHipster
+- GitHub
+- GitHub Actions
+- DevOps
+- AWS
+- Amazon ECS
+- AWS Fargate
+- Amazon ECR
+
+date: 2020-20-10T20:16:10.000Z
+title: "Deploy to ECS using GitHub Actions"
+image: https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2020/01/deployToEcsWithGithubAction.jpg
+---
+
+Building a DevOps platform nowadays can become a challenge for architecture and platform infrastructure teams, due to the integration complexity with many platforms: Cource Control, CI/CD, Artifact Repositories, Quality, SAST/SCA, Deployment, etc. Many DevOps leader believes that the challenge of DevOps is to ultimately reduce the number of platforms involved in one single pipeline.
```

Comment:
I was going to make a change to this but it feels like my changes from two days ago never got made and there are new issues with this now.  Is this a new repo? I'd rather not make the same exact comments from the first time.

Reply:
Yes, it was another pull request, I found your last comments about this part and did the asked changes.
Please let me know if you see anything else.
Thanks